docs: fix linter errors for ajs-quick-reference page.

regulas1 opened the pull request:
This addresses a part of the issue
 https://github.com/angular/angular/issues/47179

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue Number: 47179


## What is the new behavior?


## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

regulas1 committed:
docs: fix linter errors for ajs-quick-reference page. #47635

Update aio/content/guide/ajs-quick-reference.md

Update aio/content/guide/ajs-quick-reference.md

Co-Authored-By: Jessica Janiuk <72768744+jessicajaniuk@users.noreply.github.com>

regulas1 commented:
@jessicajaniuk created a new PR with the changes requested and clean git history.

jessicajaniuk commented on aio/content/guide/ajs-quick-reference.md:
```diff
@@ -66,13 +66,13 @@ The following are some of the key AngularJS built-in directives and their equiva
 | AngularJS                                                                                                                                                                                                                        | Angular |
 |:---                                                                                                                                                                                                                              |:---     |
-| <header><code>ng-hide</code></header> In AngularJS, the `ng-hide` directive shows or hides the associated HTML element based on an expression. For more information, see [ng-show][AioGuideAjsQuickReferenceTemplateDirectives]. | <header>Bind to the <code>hidden</code> property</header> In Angular, you use property binding. Angular does not have a built-in *hide* directive. For more information, s ...
+| <header><code>ng-hide</code></header> In AngularJS, the `ng-hide` directive shows or hides the associated HTML element based on an expression. For more information, see [ng-show][AioGuideAjsQuickReferenceTemplateDirectives]. | <header>Bind to the <code>hidden</code> property</header> In Angular, you use property binding, *hide* directive is not built in. For more information, see [ng-show][AioG ...
```
```suggestion
| <header><code>ng-hide</code></header> In AngularJS, the `ng-hide` directive shows or hides the associated HTML element based on an expression. For more information, see [ng-show][AioGuideAjsQuickReferenceTemplateDirectives]. | <header>Bind to the <code>hidden</code> property</header> In Angular, you use property binding. A *hide* directive is not built in. For more information, see [ng-show][AioGuideAjsQuickReferenceTemplateDirectives]. |
```

jessicajaniuk reviewed (commented):
There's a couple of syntax issues with two sentences, but otherwise this looks good. If you can update those two sentences, I can approve this.

regulas1 commented:
@jessicajaniuk updated the changes requested.

bob-watson commented on aio/content/guide/ajs-quick-reference.md:
```diff
@@ -66,13 +66,13 @@ The following are some of the key AngularJS built-in directives and their equiva
 | AngularJS                                                                                                                                                                                                                                                                                                                                                                                                     ...
 |:---                                                                                                                                                                                                                                                                                                                                                                                                           ...
-| <header><code>ng-href</code></header> <code-example hideCopy format="html" language="html"> &lt;a ng-href="{{ angularDocsUrl }}"&gt; &NewLine; &nbsp; Angular Docs &NewLine; &lt;/a&gt; </code-example> The `ng-href` directive allows AngularJS to preprocess the `href` property. `ng-href` can replace the binding expression with the appropriate URL before the browser fetches from that URL. <br /> In ...
+| <header><code>ng-href</code></header> <code-example hideCopy format="html" language="html"> &lt;a ng-href="{{ angularDocsUrl }}"&gt; &NewLine; &nbsp; Angular Docs &NewLine; &lt;/a&gt; </code-example> The `ng-href` directive allows AngularJS to preprocess the `href` property so that it can replace the binding expression with the appropriate URL before the browser fetches from that URL. <br /> In ...
```
This is a bit hard to edit in one line, but might this be clearer?
```suggestion
| <header><code>ng-href</code></header> <code-example hideCopy format="html" language="html"> &lt;a ng-href="{{ angularDocsUrl }}"&gt; &NewLine; &nbsp; Angular Docs &NewLine; &lt;/a&gt; </code-example> The `ng-href` directive allows AngularJS to preprocess the `href` property so that it can replace the binding expression with the appropriate URL before the browser fetches from that URL. <br /> In AngularJS, the `ng-href` is often used to activate a route as part of navigation. <br /> <code-example hideCopy format="html" language="html"> &lt;a ng-href="#{{ moviesHash }}"&gt; &NewLine;&nbsp; Movies &NewLine;&lt;/a&gt; </code-example> Routing is handled differently in Angular. | <header>Bind to the <code>href</code> property</header> <code-example hideCopy path="ajs-quick-reference/src/app/app.component.html" region="href"></code-example> Angular uses property binding, and does not have an `href` directive. Place the `href` property of the element in square brackets and set it to a quoted template expression. For more information see the [Property binding][AioGuidePropertyBinding] page. In Angular, `href` is no longer used for routing. Routing uses `routerLink`, as shown in the following example. <code-example hideCopy path="ajs-quick-reference/src/app/app.component.html" region="router-link"></code-example> For more information on routing, see [Defining a basic route][AioGuideRouterDefiningABasicRoute] in the [Routing & Navigation][AioGuideRouter] page. |
```

bob-watson commented on aio/content/guide/ajs-quick-reference.md:
```diff
@@ -84,7 +84,9 @@ The following are some of the key AngularJS built-in directives and their equiva
 |:---                                                                                                                                                                                                                                                                                                                                                                                                           ...
-| <header><code>ng-model</code></header> <code-example hideCopy format="html" language="html"> &lt;input ng-model="vm.favoriteHero" /&gt; </code-example> In AngularJS, the `ng-model` directive binds a form control to a property in the controller associated with the template. This provides **two-way binding** whereby changes result in the value in the view and the model being synchronized. | <head ...
+
+| <header><code>ng-model</code></header> <code-example hideCopy format="html" language="html"> &lt;input ng-model="vm.favoriteHero" /&gt; </code-example> In AngularJS, the `ng-model` directive binds a form control to a property in the controller associated with the template. This provides **two-way binding**, whereby any change made to the value in the view is synchronized with the model, and any ...
```
Comma phrases are preferred over parenthetical phrases, such as: 
```suggestion
| <header><code>ng-model</code></header> <code-example hideCopy format="html" language="html"> &lt;input ng-model="vm.favoriteHero" /&gt; </code-example> In AngularJS, the `ng-model` directive binds a form control to a property in the controller associated with the template. This provides **two-way binding**, such that any change made to the value in the view is synchronized with the model, and any change to the model is synchronized with the value in the view. | <header><code>ngModel</code></header> <code-example hideCopy path="ajs-quick-reference/src/app/movie-list.component.html" region="ngModel"></code-example> In Angular, **two-way binding** is indicated by `[()]`, referred to as a *banana in a box*. This syntax is a shortcut for defining both property binding, from the component to the view, and event binding, from the view to the component. <br /> For more information about two-way binding with `ngModel`, see the [Displaying and updating properties with `ngModel`][AioGuideBuiltInDirectivesDisplayingAndUpdatingPropertiesWithNgmodel] section of [Built-in directives][AioGuideBuiltInDirectives]. |
```

bob-watson commented on aio/content/guide/ajs-quick-reference.md:
```diff
@@ -96,13 +98,13 @@ The following are some of the key AngularJS built-in directives and their equiva
 | AngularJS                                                                                                                                                                                                                                                                                                                                                                                                     ...
 |:---                                                                                                                                                                                                                                                                                                                                                                                                           ...
-| <header><code>ng-show</code></header> <code-example hideCopy format="html" language="html"> &lt;h3 ng-show="vm.favoriteHero"&gt; &NewLine; &nbsp; Your favorite hero is: {{vm.favoriteHero}} &NewLine; &lt;/h3&gt; </code-example> In AngularJS, the `ng-show` directive shows or hides the associated DOM element, based on an expression. <br /> In this example, the `<div>` element is shown if the `favo ...
+| <header><code>ng-show</code></header> <code-example hideCopy format="html" language="html"> &lt;h3 ng-show="vm.favoriteHero"&gt; &NewLine; &nbsp; Your favorite hero is: {{vm.favoriteHero}} &NewLine; &lt;/h3&gt; </code-example> In AngularJS, the `ng-show` directive shows or hides the associated DOM element, based on an expression. <br /> In this example, the `<div>` element is shown if the `favo ...
```
```suggestion
| <header><code>ng-show</code></header> <code-example hideCopy format="html" language="html"> &lt;h3 ng-show="vm.favoriteHero"&gt; &NewLine; &nbsp; Your favorite hero is: {{vm.favoriteHero}} &NewLine; &lt;/h3&gt; </code-example> In AngularJS, the `ng-show` directive shows or hides the associated DOM element, based on an expression. <br /> In this example, the `<div>` element is shown if the `favoriteHero` variable is truthy. | <header>Bind to the <code>hidden</code> property</header> <code-example hideCopy path="ajs-quick-reference/src/app/movie-list.component.html" region="hidden"></code-example> Angular uses property binding and does not provide a *show* directive. For hiding and showing elements, bind to the HTML `hidden` property. <br /> To conditionally display an element, place the `hidden` property of the element in square brackets and set it to a quoted template expression that evaluates to the opposite of a *show* value. <br /> In this example, the `<div>` element is hidden if the `favoriteHero` variable is not truthy. <br /> For more information on property binding, see the [Property binding][AioGuidePropertyBinding] page. |
```

bob-watson commented on aio/content/guide/ajs-quick-reference.md:
```diff
@@ -192,17 +194,17 @@ The Angular code is shown using TypeScript.
 | AngularJS                                                                                                                                                                                                                                                      | Angular |
 |:---                                                                                                                                                                                                                                                            |:---     |
-| <header>Controller function</header> <code-example hideCopy format="typescript" language="typescript"> function MovieListCtrl(movieService) { &NewLine; } </code-example> In AngularJS, you write the code for the model and methods in a controller function. | <header>Component class</header> <code-example hideCopy path="ajs-quick-reference/src/app/movie-list.component.ts" region="class"></code-exa ...
+| <header>Controller function</header> <code-example hideCopy format="typescript" language="typescript"> function MovieListCtrl(movieService) { &NewLine; } </code-example> In AngularJS, you write the code for the model and methods in a controller function. | <header>Component class</header> <code-example hideCopy path="ajs-quick-reference/src/app/movie-list.component.ts" region="class"></code-exa ...
```
If this is referring to an Angular Component, maybe it should be like:
```suggestion
| <header>Controller function</header> <code-example hideCopy format="typescript" language="typescript"> function MovieListCtrl(movieService) { &NewLine; } </code-example> In AngularJS, you write the code for the model and methods in a controller function. | <header>Component class</header> <code-example hideCopy path="ajs-quick-reference/src/app/movie-list.component.ts" region="class"></code-example> In Angular, you create a component class to contain the data model and control methods. Use the TypeScript <code>export</code> keyword to export the class so that the `Component` can be imported into NgModules. <br /> For more information, see the [Components][AioGuideArchitectureComponents] section of the [Architecture Overview][AioGuideArchitecture] page. |
```

bob-watson reviewed (changes requested):
Thanks for the edits.
I suggested a few minor revisions in the comments.

regulas1 committed:
Merge branch 'angular:main' into DocumentationUpdate47179-1

regulas1 committed:
docs: fix linter errors for ajs-quick-reference page.

docs: requested changes 1

better readability and changes requested

docs: commit suggestion Update aio/content/guide/ajs-quick-reference.md

docs: commit sugggestion 3 Update aio/content/guide/ajs-quick-reference.md
Co-Authored-By: Bob Watson <104218420+bob-watson@users.noreply.github.com>

bob-watson commented:
Thanks for addressing the edits.

Unfortunately, your branch picked up a merge commit.
Angular tries to [maintain a linear commit history without merge commits](https://angular.io/guide/doc-pr-prep#why-you-rebase-your-working-branch), so you'll need to clean this up before we can merge your changes.

What we'd like to see is a single commit that's based off the current main branch with the net result of the changes you've made.

I can't give you a specific procedure to do this, but here are some resources that can help you. **I recommend that you create a new branch from this one to practice fixing this in a separate branch. This should prevent accidentally complicating this branch beyond repair.**

Here are some resources to help:

* [Rebase your working branch](https://angular.io/guide/doc-pr-prep#rebase-your-working-branch)
* [Clean up the branch](https://angular.io/guide/doc-pr-update#clean-up-the-branch)
* [StackOverflow article on removing a merge commit](https://stackoverflow.com/questions/17577409/git-remove-merge-commit-from-history) Note: read the entire topic before attempting any changes as there are several methods described.

regulas1 committed:
Merge branch 'angular:main' into DocumentationUpdate47179-1
